jib-core/src/main/java/com/google/cloud/tools/jib/Timer.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib;
+
+import java.io.Closeable;
+import java.util.Stack;
+
+/**
+ * Times execution intervals. This is only for testing purposes and will be removed before the first
```

Comment:
is there an issue for this?

Reply:
Filed https://github.com/google/jib/issues/30